README.md

```diff
@@ -246,9 +246,10 @@ conn.on(:stream) do |stream|
     }
 
     # initiate server push stream
+    push_stream = nil
     stream.promise(head) do |push|
-      push.headers({ ... })
-      push.data(...)
```

Comment:
That's what .promise does above.. The latter call is to demonstrate that you can send additional headers if you want, before you proceed with body.